cranelift-codegen/Cargo.toml

```diff
@@ -23,6 +23,7 @@ target-lexicon = "0.8.1"
 log = { version = "0.4.6", default-features = false }
 serde = { version = "1.0.94", features = ["derive"], optional = true }
 smallvec = { version = "0.6.10" }
+byteorder = "1.3.2"
```

Comment:
Can you add `default-features = false` here, to enable support for `no_std` builds? (as described in [the README](https://crates.io/crates/byteorder))

`no_std` isn't a high priority right now, but it may be in the future, so it's useful to keep an eye out for it when adding new dependencies.

Reply:
I'll add.